File: dayTwo/java/AfterBreakTask/Java/GradeScore.java

import java.util.Scanner;

public class GradeScore{

public static int grades(int average){
if(90<=average <=100){
return ("A");
}
   
else if(80<=average <=90){
System.out.print("B");

}
       
else if (70<=average <=80){

System.out.print("c");

}
        
else if (60<=average <=70){

System.out.print("D");

}
    else{
       System.out.print("F");

}
return (average) / 3;

}

public static void main(String[] args){

Scanner ageCollector= new Scanner (System.in);

	System.out.print("The First Score: ");

		int scoreOne = ageCollector.nextInt();

	System.out.print("The Second Score: ");
		
		int scoreTwo = ageCollector.nextInt();
		
		System.out.print("The Third Score: ");
		
		int scoreThree = ageCollector.nextInt();
 int average = grades(average)/ 3;

System.out.println("The grade is: " + average + " years");


}
}

File: dayTwo/python/AfterBreakTask/Python/grade_score.py
def grade(average):
    if 90<=average <=100:
        return "A"
    elif 80<=average <=90:
        return "B"
    elif 70<=average <=80:
        return "C"
    elif 60<=average <=70:
        return "D"
    else:
        return "F"


def main():
    score_one = float(input("First Score: "))
    score_two = float(input("Second Score: "))
    score_three = float(input("Third Score: "))

    average = (score_one + score_two + score_three) / 3

    print(grade(average))
    print("The Average is",average)
    
main()
